README.md

```diff
@@ -4,87 +4,92 @@
+To view all releases, [**click here**](https://github.com/neon-nyan/CollapseLauncher/releases).
 
-# Prequesties for Ready-to-use build
-- OS: **Windows 10 1809 Update (build 17763)** and later or **Windows 11 (Any builds)**
+# System Requirements for Ready-to-use build
+- OS: **Windows 10 1809 Update (build 17763)** or later / **Windows 11 (Any builds)**
 - Architecture: **x64/AMD64**
 - Internet Access: **Yes**
 
 ***
 
-# Prequesties for self-build/development environment
-To develop and build it for yourself, you need to install [**.NET Core 7 SDK (7.0.100 or later)**](https://dotnet.microsoft.com/en-us/download/dotnet/7.0), [**WinUI 3 (WindowsAppSDK 1.2 Stable Runtime)**](https://aka.ms/windowsappsdk/1.2/latest/windowsappruntimeinstall-x64.exe) and **Windows 10 SDK (10.0.19043.0 or later)** via Visual Studio Installer.
+# Prerequisites for Building Locally/Development
+*Collapse* is presently powered by .NET 7 and as such, the packages listed below are required to create a local and development build of the launcher. Furthermore, *Collapse* uses many submodules and packages outside of this, which will automatically be loaded when the user sets up a local environment of the application.
+- .NET: [**.NET Core 7 SDK (7.0.100 or later)**](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
+- WinUI 3: [**WinUI 3 (WindowsAppSDK 1.2 Stable Runtime)**](https://aka.ms/windowsappsdk/1.2/latest/windowsappruntimeinstall-x64.exe)
+- WindowsAppSDK 1.2 Stable C# Extension: [**Visual Studio 2022 C# Extension**](https://aka.ms/windowsappsdk/1.2/latest/WindowsAppSDK.Cs.Extension.Dev17.Standalone.vsix)
+- **Windows 10 SDK (10.0.19043.0 or later)** via Visual Studio Installer
 
 > **Note**:
 > 
-> Starting from November 13<sup>rd</sup> 2022, you must have Visual Studio 2022 installed on your computer due to minimum requirement of WinUI 3 1.2 Stable release.
+> Starting from November 13<sup>rd</sup> 2022, you <b>must</b> have Visual Studio 2022 installed on your computer due to the updated minimum system requirement of `WinUI 3 1.2 Stable`.
 > 
-> Using lower Visual Studio version (like VS2019) is possible, but you have to downgrade the **WindowsAppSDK** via NuGet to **WindowsAppSDK 1.1.5** or **WindowsAppSDK 1.2-preview2** before building.
-
-You may also need **Visual Studio 2022** or later with **WindowsAppSDK 1.2 Stable C# Extension** installed.
-Please refer to this link to download the extension for your Visual Studio version
-- [**Visual Studio 2022 C# Extension**](https://aka.ms/windowsappsdk/1.2/latest/WindowsAppSDK.Cs.Extension.Dev17.Standalone.vsix)
+> Using a lower Visual Studio version (like VS2019) is possible, but you is not recommended as you need to downgrade **WindowsAppSDK** via *NuGet* to **WindowsAppSDK 1.1.5** or **WindowsAppSDK 1.2-preview2** before building. This has an increased risk of breaking the application and as such, minimal support will be provided for this method. **This is not recommended for beginner users.**
```

Comment:
I think the part ``but "you is" not recommended as you need to downgrade`` must be changed to ``but "it is" not recommended as you need to downgrade``